src/Sylius/Bundle/ApiBundle/Swagger/ProductSlugDocumentationNormalizer.php

```diff
@@ -38,6 +38,8 @@ public function normalize($object, $format = null, array $context = [])
         foreach ($params as $index => $param) {
             if ($param['name'] === 'code') {
                 unset($docs['paths'][self::PRODUCT_SLUG_PATH]['get']['parameters'][$index]);
+                // reset key index after unset
+                $docs['paths'][self::PRODUCT_SLUG_PATH]['get']['parameters'] = array_values($docs['paths'][self::PRODUCT_SLUG_PATH]['get']['parameters']);
```

Comment:
Could you move it out of the loop to line ~45?

Reply:
It's done !